app/controllers/concerns/issues/filters.rb

```diff
@@ -16,7 +16,8 @@ module Issues::Filters
     :comment,
     :key,
     :created_at,
-    :scheduled_at
+    :scheduled_at,
+    { data_query_keys: {} }
```

Comment:
Le pondría solo el nombre del campo:
```suggestion
    { canonical_data: {} }
```

Reply:
perfecto